### DISCUSSION TITLE
Disdaqs (dummy scan) removal

### DISCUSSION THREAD

**Discussion by cchaswell** [HALFpipe/HALFpipe#446] [Q&A]
Is there a way to set how many timepoints you want removed for HALFpipe, or does it need to be done prior to running? I know fmriprep allows a dummyscan option but wasn't sure if it is implemented in HALFpipe. Thanks!

**Comment by HippocampusGirl** [comment]
Hi @cchaswell,

The number of time points to remove indeed needs to be setup before running. The user interface will prompt you for that number. In case you want to modify it later, you can also set the number of dummy scans in the `spec.json` file.

For that, change
```json
        "dummy_scans": 0,
```
to
```json
        "dummy_scans": 10,
```
or whichever number is correct :-)
